src/adapter/src/catalog/storage.rs

```diff
@@ -13,6 +13,7 @@ use std::iter::once;
 
 use bytes::BufMut;
 use itertools::max;
+use mz_controller::ConcreteComputeInstanceReplicaConfig;
```

Comment:
This should be down with the other mz_ imports, not mixed with the third-party imports.